content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -194,6 +194,54 @@ A link to the example repository used is also provided in the link:https://youtu
 
 video::eV7roTXrEqg[youtube,width=800,height=420]
 
+=== Statically loading evaluated versions of libraries
+
+As of currently unreleased version of the _Pipeline: Shared Groovy Libraries_ plugin,
+with features proposed in
+link:https://github.com/jenkinsci/pipeline-groovy-lib-plugin/pull/19[PR #19]
+(link:https://issues.jenkins.io/browse/JENKINS-69731[issue JENKINS-69731])
```

Comment:
Thanks for phrasing it that way to make it clear to reviewers like me that the feature is not yet released.  Very nice technique!
```suggestion
```

Reply:
I suppose you did not mean removing these lines, just yet? :)

(If this PR is indeed only merged after the feature release, this should get rephrased similar to `library` step description)